code/modules/clothing/glasses/hud.dm

```diff
@@ -118,7 +96,6 @@
 	name = "diagnostic HUD"
 	desc = "A heads-up display capable of analyzing the integrity and status of robotics and exosuits."
 	icon_state = "diagnostichud"
-	hud_types = list(DATA_HUD_DIAGNOSTIC_BASIC)
 	clothing_traits = list(TRAIT_DIAGNOSTIC_HUD)
```

Comment:
Diagnostic huds, currently, give `DATA_HUD_DIAGNOSTIC_BASIC`
In this PR, `TRAIT_DIAGNOSTIC_HUD` gives `DATA_HUD_DIAGNOSTIC_ADVANCED`
So this is an error. (The only error I noticed in the PR though) 

But the only difference between the two is the advanced hud shows but paths. Maybe we should just separate bot paths into its own hud (that things like the AI and observers use)?